### ISSUE TITLE
wallet_results.php has inconsistent values in summary table

### ISSUE THREAD

**Issue by auswonk** [tpruvot/yiimp#43]
The earnings summary on the wallet page shows an incorrect value in the detailed per-coin table (when visible).

![wallet](https://cloud.githubusercontent.com/assets/19761222/19219576/77957d7e-8e62-11e6-8cde-6e6a34391a68.png)

The detailed results do not seem to include either the pool fee or the exchange fees.  Perhaps the data is being stored in the memcache before fees are applied?

The summary results seem to come directly from SQL, which has the post-fee value stored:

```
mysql> select * from earnings where userid=1;
+-------+--------+--------+---------+-------------+-----------------+-------------+--------+-------------+
| id    | userid | coinid | blockid | create_time | amount          | price       | status | mature_time |
+-------+--------+--------+---------+-------------+-----------------+-------------+--------+-------------+
| 17494 |      1 |    326 |   16500 |  1476000782 | 99.563730586027 | 0.000000045 |      0 |        NULL |
+-------+--------+--------+---------+-------------+-----------------+-------------+--------+-------------+
1 row in set (0.00 sec)
```
99.563730586027 * 0.000000045 = 0.00000448

**Comment by tpruvot** [comment]
Indeed, should be fixed now in 3db9327372a62814325d20c2dd92ab08554a9999

**Comment by auswonk** [comment]
Patch applied and confirmed fixed.  Thanks!
